src/components/textBlock/TextBlock.js

```diff
@@ -31,14 +31,13 @@ export class TextBlock extends Component {
       )
     } else {
       fieldHTML = (
-        <textarea
-            name="text"
+        <input
```

Comment:
and in that part http://i.imgur.com/Fo2NPAw.png it should be a textarea, so your component requires some changes (it can be and input, and textarea)